web/src/test/java/org/springframework/security/web/server/authentication/SwitchUserWebFilterTests.java

```diff
@@ -183,11 +179,12 @@ public void switchUserWhenUsernameIsMissingThenThrowException() {
 				.from(MockServerHttpRequest.post("/login/impersonate"));
 		final WebFilterChain chain = mock(WebFilterChain.class);
 		final SecurityContextImpl securityContext = new SecurityContextImpl(mock(Authentication.class));
-		this.exceptionRule.expect(IllegalArgumentException.class);
-		this.exceptionRule.expectMessage("The userName can not be null.");
-		this.switchUserWebFilter.filter(exchange, chain)
-				.subscriberContext(ReactiveSecurityContextHolder.withSecurityContext(Mono.just(securityContext)))
-				.block();
+		assertThatIllegalArgumentException()
+				.isThrownBy(() -> this.switchUserWebFilter.filter(exchange, chain)
```

Comment:
This is another spot where I'm confused about the line wrapping. Compare this to line 220 and 221 where the content is identical, but separated across two lines.

Reply:
This is related to the length of the contained line. Extracting the holder to a variable helps quite a bit. I'll push a fix.